### PR TITLE
Final Commits

### PR DESCRIPTION
**Pull request by IsmailChangezi**
#### All of the requirements below have been met

Interactions
Splash screen

A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.
Sign up and log in pages

The user should be able to register in the app with full name, email and password (all mandatory).
The user can log into the app using email and password.
If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).
Home page (categories page)

When the user logs in, they are presented with the categories page.
For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
Transactions page

For a given category, the list of transactions is presented, ordered by the most recent.
At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
When the user clicks on the "Back" button (<), the user navigates to the home page.
"Add a new category" page

The user fills out a form to create a new category, indicating their name and icon (both mandatory).
The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.
When the user clicks on the "Back" button (<), the user navigates to the home page.
"Add a new transaction" page

The user fills out a form to create a new transaction with:
name (mandatory)
amount (mandatory)
categories (mandatory at least one)
The user click (or taps) the "Save" button to create the new transaction, and is taken to the transactions page for that category.
When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.
Testing requirements
Create unit and integration tests for all the most important components of your RoR application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,156 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please leave a start to support
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for this great learning experience !
+
```
Comment (algerina) [inline README.md]: - [x] It is required that he README contains an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,156 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Video Explanation
+This is the link to the video explaination
+
+https://www.loom.com/share/My-Project-Journey-0daf09dd9d7242559ce9247b87e25aa8?sid=058c64bf-1f3c-4da9-85a3-62b53dcca18e
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```
Comment (algerina) [inline README.md]: - [ ] Kindly deploy the application using Render, Netlify or GitHub Pages, you should have a link to the deployed project  in the README.

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<section class="bold main-text">SnapScan</section>
+
+<section class="login-btns container">
+    <%= button_to "LOG IN" , new_user_session_path, class: "login-btn" %>
+    <%= link_to "SIGN UP" , new_user_registration_path, class: "btn btn-primary" %>
+</section>
```
Comment (algerina) [inline app/views/home/index.html.erb:6]: - [ ] I will suggest adding some space between the two buttons to much the design.

File: app/assets/stylesheets/application.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+}
+
+.login-btns * {
+  width: 100%;
+  height: 40px;
+}
+
```
Comment (algerina) [inline app/assets/stylesheets/application.css:34]: - [ ] Well done styling your project however you should follow the design and using the same colors for the buttons and the header.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @IsmailChangezi ,

Great job so far! 

May I highlight the following: 
-  Your app is almost working well ✔
- The video presentation is descriptive✔️



## CHANGES REQUIRED:muscle::clap:


Check the comments under the review.



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Comment by omar-muhamad** [comment]
# Invalid ❌

Hi @IsmailChangezi,

#### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

#### Your Code Review Request is marked as invalid for this reason:

- [ ] [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) workflow is not correctly used, Kindly make this Pull Request to compare changes from the feature branch to the development(dev) branch. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Pull request closed without merging**